src/main/java/seedu/eventfulnus/logic/Messages.java

```diff
@@ -26,132 +26,6 @@ public class Messages {
             EventfulNUS allows you to manage a list of people and events, for easier event planning for\s
             NUS Inter-Faculty Games.
 
-            These are the available commands:
```

Comment:
Great that you removed the help pop-up message feature because it was simply too much text